### PR TITLE
Generate model name with UpperCamelCase

### LINKED ISSUE #5: Multi-words model name is incorrect
## WHY

If table name consists of multiple words, pq2gorm generates model name with snake_case.

For example, from `post_comments` table is converted to `Post_comment` model.

``` go
type Post_comment struct {

}
```
## WHAT

Multi-words table name should converted to UpperCamelCase model name.

``` go
type PostComment struct {

}
```

### PR DESCRIPTION
**Pull request by dtan4**
Close #5 
## WHY

Multi-words table name is converted to snake_case table name. This is incorrect as Golang struct name.
## WHAT

Convert multi-words table name to UpperCamelCase table name.

### CODE REVIEW

File: generate.go [hunk 1]
```diff
@@ -49,9 +50,13 @@ func genJSON(columnName, columnDefault string, primaryKeys map[string]bool) (jso
...
 // Singlarlize table name and upper initial character
 func gormTableName(s string) string {
-	s = strings.ToLower(s)
-	s = inflector.Singularize(s)
-	return strings.Title(s)
+	var tableName string
+
+	tableName = strings.ToLower(s)
+	tableName = inflector.Singularize(tableName)
+	tableName = snaker.SnakeToCamel(tableName)
```
Comment (koudaiii) [inline generate.go:57]: `ToLower -> Singularize -> SnakeToCamel` is very simple 👀

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,14 @@
 sudo: required
 services:
   - docker
+language: go
+go:
+  - '1.7'
```
Comment (koudaiii) [inline .travis.yml:7]: I think that should add `1.5` and `1.6` 👀 

because of.
https://github.com/wantedly/pq2gorm#how-to-build-and-install

Comment (dtan4) [reply .travis.yml:7]: https://github.com/wantedly/pq2gorm/pull/6/commits/5e1f1138345b4ec2aca874aaabe7865fadccee2e Added Go 1.6 environment.

As far as I read [Release History - The Go Programming Language](https://golang.org/doc/devel/release.html#policy), Go 1.6.2 and earlier are no longer supported officially. So now we can drop Go 1.5 support, I think.

Comment (koudaiii) [reply .travis.yml:7]: 🙆

**Comment by koudaiii** [comment]
LGTM!!!

**Pull request merged**